### PR TITLE
Result

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
Result page added.

### CODE REVIEW

**Review by parthgabani777 — APPROVED** [review]
Looks Good!

File: pages/result.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quizgy | Homepage</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
+    <link rel="stylesheet" href="https://lilac-ui.netlify.app/css/components.css">
+    <link rel="stylesheet" href="/css/result.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Akaya+Telivigala&display=swap');
+    </style>
```
Comment (darshandamre) [inline pages/result.html]: instead of this you can use link tag

Comment (darshandamre) [reply pages/result.html]: rest of the things look good

Comment (muskan-gupta-0803) [reply pages/result.html]: okay, thanks for reviewing.

**Pull request merged**